changelog/2021-03-09T10_44_18+01_00_rename_manifest

```diff
@@ -0,0 +1 @@
+CHANGED: '$topEntity.manifest' has been renamed to 'manifest' in order to make it easier for external tools to find it.
```

Comment:
Do we plan to support external tools doing this?
Then we might have to think carefully about them file format.

Reply:
Yes, I would really like that. Gergo's `clashilator` is already using it, and I'd like to use it in one of our internal projects (preferably from Python). It would be great if we could remove `-fclash-edalize` too, and just make a pypi package that can read manifest files.

The path there isn't really clear to me. At the very least, we should use some language-independent format (JSON?). I think we should then dogfeed it and see where we end up. But that's post-1.6 for sure.

Reply:
I'm going to pull the patch out and put it in a separate PR.